Index: machette/machette/__init__.py
===================================================================
--- machette.orig/machette/__init__.py	2010-07-17 14:24:42.000000000 +0200
+++ machette/machette/__init__.py	2010-07-17 14:24:54.000000000 +0200
@@ -444,7 +444,7 @@
 
 			# Load information from external files
 			try:
-				fd = open(os.path.join(DATA_DIR, 'LICENSE'))
+				fd = open('/usr/share/common-licenses/GPL-3')
 				self.about_dialog.set_license(fd.read())
 				fd.close()
 			except IOError, e:
Index: machette/setup.py
===================================================================
--- machette.orig/setup.py	2010-07-17 14:24:58.000000000 +0200
+++ machette/setup.py	2010-07-17 14:25:16.000000000 +0200
@@ -220,7 +220,7 @@
 		'machette.ui',
 	],
 	data_files=[
-		( 'share/' + __cmdname__, [ 'AUTHORS', 'ChangeLog', 'LICENSE', 'README', 'TRANSLATORS' ] ),
+		( 'share/' + __cmdname__, [ 'AUTHORS', 'ChangeLog', 'README', 'TRANSLATORS' ] ),
 		( 'share/' + __cmdname__ + '/ui', [ 'ui/main.ui', 'ui/pref.ui' ] ),
 		( 'share/' + __cmdname__ + '/ui/module', [ 'ui/module/group.ui', 'ui/module/replace.ui', 'ui/module/split.ui' ] ),
 	],
